### DISCUSSION TITLE
How about bring some discussions to dotnet-discord

### DISCUSSION THREAD

**Discussion by lindexi** [dotnet/wpf#7454] [Ideas]
The dotnet discord: https://aka.ms/dotnet-discord

How about bring some discussions to dotnet-discord?

Any advantage? Maybe...

**Comment by gfoidl** [comment]
> Any advantage? 

Another channel to spread informations / discussions around. Similar to ![](https://imgs.xkcd.com/comics/standards.png) 😉

Seriously: there's already issues and discussion for that. So why another location for a similar thing?
